doc/user_guide/example-udf-script/execute.sql

```diff
@@ -0,0 +1,20 @@
+EXECUTE SCRIPT "AAF_DB_SCHEMA"."AAF_RUN_QUERY_HANDLER"('{
+    "query_handler": {
+        "factory_class": {
+            "module": "example_module",
+            "name": "ExampleQueryHandlerFactory"
+        },
+        "parameter": "bla-bla",
+        "udf": {
+            "schema": "EXAMPLE_SCHEMA",
+            "name": "MY_QUERY_HANDLER_UDF"
```

Comment:
if we already run with the word example, should we use it everywhere (half a joke)

Reply:
Actually, during my experiments, I switched from MY_SCHEMA to MY_EXAMPLE to avoid false positive test results due to my test database being already partially initialized.

Should I replace the name `MY_QUERY_HANDLER_UDF` by `EXAMPLE_QUERY_HANDLER_UDF`?